dotnet/src/webdriver/DriverProcessStartedEventArgs.cs

```diff
@@ -41,31 +40,10 @@ public DriverProcessStartedEventArgs(Process driverProcess)
         }
 
         this.ProcessId = driverProcess.Id;
-        if (driverProcess.StartInfo.RedirectStandardOutput && !driverProcess.StartInfo.UseShellExecute)
```

Comment:
This might be to aggressive but since we don't want to do sync reads, we might delete it. Or deprecate first?

Reply:
I did a quick search in GitHub and other places for `DriverProcessStartedEventArgs` and did not see mentions.

Reply:
What do you mean by leaving this thread unresolved? What should we wait for?